### ISSUE TITLE
dials.import with image_range broken

### ISSUE THREAD

**Issue by Anthchirp** [dials/dials#337]
```
$ dials.import /dls/i19-1/data/2017/cm16760-2/50micron_cryst1/50micron_cryst1_05_00001.cbf geometry.scan.image_range=1,900 geometry.scan.extrapolate_scan=True
DIALS 1.dev.1307-g7778ce0
The following parameters have been modified:

input {
  datablock = <image files>
}
geometry {
  scan {
    image_range = 1 900
    extrapolate_scan = True
  }
}


Applying input geometry in the following order:
  1. Manual geometry

Traceback (most recent call last):
  File "/scratch/wra62962/files/dials/build/../modules/dials/command_line/dials_import.py", line 739, in <module>
    halraiser(e)
  File "/scratch/wra62962/files/dials/build/../modules/dials/command_line/dials_import.py", line 737, in <module>
    script.run()
  File "/scratch/wra62962/files/dials/build/../modules/dials/command_line/dials_import.py", line 630, in run
    datablocks = map(metadata_updater, datablock_importer())
  File "/scratch/wra62962/files/dials/build/../modules/dials/command_line/dials_import.py", line 460, in __call__
    imageset = updater(imageset)
  File "/scratch/wra62962/files/dials/build/../modules/dials/command_line/dials_import.py", line 321, in __call__
    assert i1 <= j1, ("%d should be <= %d" % (i1, j1))
AssertionError: Please report this error to dials-support@lists.sourceforge.net: 900 should be <= 1
```

In the past:
```
$ dials.import /dls/i19-1/data/2017/cm16760-2/50micron_cryst1/50micron_cryst1_05_00001.cbf geometry.scan.image_range=1,900 geometry.scan.extrapolate_scan=True
DIALS 1.4.5-g37e56cb3-release
The following parameters have been modified:

input {
  datablock = <image files>
}
geometry {
  scan {
    image_range = 1 900
    extrapolate_scan = True
  }
}

--------------------------------------------------------------------------------
  format: <class 'dxtbx.format.FormatCBFFullPilatus.FormatCBFFullPilatus'>
  num images: 900
  num sweeps: 1
  num stills: 0
--------------------------------------------------------------------------------
Writing datablocks to datablock.json
```

**Comment by jmp1985** [comment]
Fixed

**Comment by Anthchirp** [comment]
Fix for this didn't work on 1.5 branch:
```
$ libtbx.python ~/local/dials/modules/dials/test/command_line/tst_import.py 
OK
OK
OK
Traceback (most recent call last):
  File "/home/wra62962/local/dials/modules/dials/test/command_line/tst_import.py", line 216, in <module>
    test.run()
  File "/home/wra62962/local/dials/modules/dials/test/command_line/tst_import.py", line 20, in run
    self.tst_override_geometry()
  File "/home/wra62962/local/dials/modules/dials/test/command_line/tst_import.py", line 119, in tst_override_geometry
    easy_run.fully_buffered(cmd).raise_if_errors()
  File "/scratch/wra62962/files/dials/modules/cctbx_project/libtbx/easy_run.py", line 39, in raise_if_errors
    raise Error(msg)
RuntimeError: child process stderr output:
  command: 'dials.import /scratch/wra62962/files/dials/modules/dials_regression/centroid_test_data/centroid_0001.cbf /scratch/wra62962/files/dials/modules/dials_regression/centroid_test_data/centroid_0005.cbf /scratch/wra62962/files/dials/modules/dials_regression/centroid_test_data/centroid_0007.cbf /scratch/wra62962/files/dials/modules/dials_regression/centroid_test_data/centroid_0006.cbf /scratch/wra62962/files/dials/modules/dials_regression/centroid_test_data/centroid_0004.cbf /scratch/wra62962/files/dials/modules/dials_regression/centroid_test_data/centroid_0002.cbf /scratch/wra62962/files/dials/modules/dials_regression/centroid_test_data/centroid_0008.cbf /scratch/wra62962/files/dials/modules/dials_regression/centroid_test_data/centroid_0003.cbf /scratch/wra62962/files/dials/modules/dials_regression/centroid_test_data/centroid_0009.cbf geometry.phil output.datablock=override_geometry.json'
  Traceback (most recent call last):
    File "/scratch/wra62962/files/dials/build/../modules/dials/command_line/dials_import.py", line 734, in <module>
      halraiser(e)
    File "/scratch/wra62962/files/dials/build/../modules/dials/command_line/dials_import.py", line 732, in <module>
      script.run()
    File "/scratch/wra62962/files/dials/build/../modules/dials/command_line/dials_import.py", line 625, in run
      datablocks = map(metadata_updater, datablock_importer())
    File "/scratch/wra62962/files/dials/build/../modules/dials/command_line/dials_import.py", line 455, in __call__
      imageset = updater(imageset)
    File "/scratch/wra62962/files/dials/build/../modules/dials/command_line/dials_import.py", line 323, in __call__
      imageset.set_scan(scan)
    File "/scratch/wra62962/files/dials/modules/cctbx_project/dxtbx/imageset.py", line 1220, in set_scan
      assert(scan.get_num_images() == (self._indices[-1] - self._indices[0] + 1))
  AssertionError: Please report this error to dials-support@lists.sourceforge.net:
```

**Comment by Anthchirp** [comment]
[If you want to run the test on your machine](https://github.com/dials/dials/wiki/How-do-I-test-the-upcoming-point-release)

**Comment by Anthchirp** [comment]
Fixed on 1.5 branch by https://github.com/dials/dials/commit/4b41e71659ebb84a19cbd7d21f1c4c8434faaf42

**Comment by jmp1985** [comment]
Hi Markus,

You need to update dxtbx as well to get this to work.

James

________________________________

**Comment by Anthchirp** [comment]
1.5 tests are using the dials-next branches of cctbx/dxtbx and dials. If fix in one repo depends on change in another repo then both need to be fixed on their respective dials-next branches.

As far as I can tell both repositories got updated here, but dxtbx has already diverged far enough that the release fix no longer worked on the 1.5 branch.
